File: Web/Vaishnavi-Singh20002_VaishnaviSingh_2125ec1060_02/Patterns/pattern3.cpp
#include <iostream>
using namespace std;

int main(){
    int n;
    cin >> n;
    for(int i=1;i<=n;i++)
    {
        for(int j=i;j<n;j++){
            cout<<"\t";
        }
        for(int j = 1;j <= i; j++){
            cout<<"*\t";
        }
        cout<<endl;
    }
    
    //write your code here
    
}

File: Android/Cyclotron17_SambhrantTiwari_2125csai1011_2/Pattern/Pattern3.java
import java.util.*;

public class Main {

    public static void main(String[] args) {
        Scanner scn = new Scanner(System.in);

        // write ur code here

        int n=scn.nextInt();
        for(int i=1;i<=n;i++)
        {
            for(int s=n-i;s>=1;s--)
            {
                System.out.print("\t");
            }
            for(int j=1;j<=i;j++)
            {
                System.out.print("*\t");
            }
                System.out.println();

        }
    }
}